deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/RegisterManager.java

```diff
@@ -8,6 +8,8 @@
 import com.microsoft.azure.sdk.iot.deps.serializer.ConfigurationContentParser;
```

Comment:
Questions : 1. Where is this verified to be working?
2. Will e2e be another PR? Is the task captured?

Reply:
Verified using a sample.
I added e2e task for this PBI, thanks for reminding